src/main/java/org/mitre/synthea/export/BFDExportBuilder.java

```diff
@@ -0,0 +1,167 @@
+package org.mitre.synthea.export;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.util.HashMap;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.function.Function;
+
+
+import com.opencsv.bean.CsvToBean;
+import com.opencsv.bean.CsvToBeanBuilder;
+
+/**
+ * Export configuration class for BFD
+ */
+public class BFDExportBuilder {
+
+  enum ExportConfigType {
+    BENEFICIARY,
+    BENEFICIARY_HISTORY,
+    CARRIER,
+    // DME,
+    INPATIENT,
+    // HHA,
+    // HOSPICE,
+    // MEDICARE_BENEFICIARY_ID,
+    OUTPATIENT,
+    // PDE,
+    PRESCRIPTION,
+    // SNF,
+  }
+  
+
+  private File configFile;
+  private List<BFDExportConfigEntry> allConfigs = null;
+  
+  private List<BFDExportConfigEntry> beneficiaryConfigs = new ArrayList<BFDExportConfigEntry>();
+  private List<BFDExportConfigEntry> beneficiaryHistoryConfigs = new ArrayList<BFDExportConfigEntry>();
+  private List<BFDExportConfigEntry> carrierConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> dmeConfigs = new ArrayList<BFDExportConfigEntry>();
+  private List<BFDExportConfigEntry> inpatientConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> hhaConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> hospiceConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> medicareBeneficiaryIdConfigs = new ArrayList<BFDExportConfigEntry>();
+  private List<BFDExportConfigEntry> outpatientConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> pdeConfigs = new ArrayList<BFDExportConfigEntry>();
+  private List<BFDExportConfigEntry> prescriptionConfigs = new ArrayList<BFDExportConfigEntry>();
+  // private List<BFDExportConfigEntry> snfConfigs = new ArrayList<BFDExportConfigEntry>();
+  
+  /** constructor */
+  public BFDExportBuilder() {
+    this.configFile = new File( "src/main/resources/exporters/cms_field_values.tsv" );
```

Comment:
Should probably get this as a resource. This won't work in the JAR release.

Reply:
> There is stuff in here I'm confused by, but it mostly makes sense. I didn't run it through checkStyle, but I'm sure it will have a lot of complaints.
> 
> As far as I can tell, all it does now is add fixed values -- correct?

In this PR, that is correct.  In the new work in hk_5_bb_export branch, it does distributions and other things (like encounter dates as a start)